.github/workflows/generate-format.yml

```diff
@@ -4,6 +4,10 @@ on:
   workflow_dispatch:
   push:
     branches: [main]
+    paths:
+      - "styles/**/*"
+      - "scripts/generate/*.ts"
```

Comment:
This should be any file in the `scripts/generate` directory right? This doesn't include the templates, for instance.